x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/rules/QueryRulesIndexServiceTests.java

```diff
@@ -41,6 +41,8 @@
 
 public class QueryRulesIndexServiceTests extends ESSingleNodeTestCase {
 
+    private static final int REQUEST_TIMEOUT = 10;
```

Comment:
Nit: consider renaming to `REQUEST_TIMEOUT_SECONDS` to indicate the time unit.
Or use `TimeValue` to embed the unit, and access it with `timeValue.timeUnit()`.
But since this is a test class, it's not a big deal if we leave it like this.

Reply:
I'm also wondering if the boilerplate code for managing the latch can be simplified or extracted into a private method with type parameters. Again, doesn't have to be done in this PR.

Reply:
Thanks! I renamed the variable, I'll leave the refactoring for a subsequent PR.